app/src/main/java/com/kickstarter/libs/utils/ComparatorUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.kickstarter.libs.utils;
+
+
+import android.support.annotation.NonNull;
+
+import java.util.Comparator;
+
+public final class ComparatorUtils {
+
+  public static final class IntegerComparator implements Comparator<Integer> {
+    /*
+    * IMPORTANT: this sorts in descending order.
+     */
+    @Override
+    public int compare(final @NonNull Integer x,final @NonNull Integer y) {
+      return (x < y) ? 1 : ((x.intValue() == y.intValue()) ? 0 : -1);
+    }
+  }
+}
```

Comment:
@luoser def think changing it here is a good call - we can just specify that this is a descending compare (maybe we should change the name?) and make a new one if we need ascending compare somewhere.